src/drivers/express/ExpressRouteDriver.ts

```diff
@@ -1,9 +1,9 @@
 import * as express from 'express';
 import { Router } from 'express';
-import * as proxy from 'express-http-proxy';
-import { ExpressResponder } from '../drivers';
+// tslint:disable-next-line:no-require-imports
+import proxy = require('express-http-proxy');
```

Comment:
Why is this done via a require import?

Reply:
:confused: I honestly do not remember. It's been reverted :ok_hand: